composer.json

```diff
@@ -6,11 +6,12 @@
 
     , "require": {
           "php": "^5.5|^7.0"
-        , "illuminate/database": "~5.2.0"
-        , "symfony/dependency-injection": "^2.6|^3.0"
+        , "illuminate/database": "^5.0.0 <5.3"
+        , "symfony/dependency-injection": "^2.8|^3.0"
```

Comment:
Agree with Symfony version bump. Let's keep the PHP version as it is now (I don't need to use PHP 5.6 features, this small bundle won't push any server to upgrade their PHP version and illuminate 5.2 still supports PHP 5.5).